packages/grafana-ui/src/components/FileUpload/FileUpload.tsx

```diff
@@ -44,11 +44,12 @@ export const FileUpload: FC<Props> = ({
 
   return (
     <>
-      <label className={cx(style.button, className)}>
+      <label role="button" className={cx(style.button, className)}>
         <Icon name="upload" className={style.icon} />
         {children}
         <input
           type="file"
+          data-testid="fileUpload"
```

Comment:
🤔 frequently we add these tests names into [packages/grafana-e2e-selectors/src/selectors/components.ts](https://github.com/grafana/grafana/pull/47497/files#diff-3a3d6d210a1aac4833b90c424ef71bfaffe3b02887421523e19687ade3a71006)